Makefile

```diff
@@ -12,7 +14,10 @@ help: ## Display this help text
 all: build test
 
 build: ## Build docker image
-	@docker build --file Dockerfile --tag=${IMAGE}:${VERSION} .
+	@echo "JAVA_AGENT_BRANCH=${JAVA_AGENT_BRANCH} JAVA_AGENT_REPO=${JAVA_AGENT_REPO}"
```

Comment:
This will help to see the dynamic values:

![image](https://user-images.githubusercontent.com/2871786/74243893-75218780-4cd8-11ea-8411-a24cfc4d8b05.png)


Otherwise, docker when using the ARGS does not show the new values but the defaulted ones.